tests/server/signup.test.js

```diff
@@ -0,0 +1,63 @@
+import {describe, expect, test, beforeAll, afterAll, beforeEach} from '@jest/globals';
```

Comment:
you shouldn't actually need to import these

Reply:
Resolved from 5f6151751a2ddc72e09c0ad664d49e38a503db96.